samples/meetings-visual-indicator/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.microsoft.teams.visualindicator",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://microsoft.com",
+    "privacyUrl": "https://microsoft.com/privacy",
+    "termsOfUseUrl": "https://microsoft.com/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meetings-Visual-Indicator",
+    "full": "Show Visual Indicator In Teams Meetings"
+  },
+  "description": {
+    "short": "Visual Indicator In-Meeting Notification demo",
+    "full": "This sample app shows how to send visual indicator in Teams meetings using targeted notifications API."
+  },
+  "accentColor": "#FFFFFF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://<<Valid-Domain>>/configure.html",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context": [
+        "meetingSidePanel"
+      ]
+    }
+  ],
+  "bots": [
+    {
+      "botId": "<<YOUR-MICROSOFT-APP-ID>>",
+      "scopes": [
+        "personal",
```

Comment:
Nit: as this feature is only supported in a meeting (group chat), maybe we can remove personal & teams for clarity?

Reply:
Yes, we have removed personal and teams from manifest.
Thanks!